PWGDQ/Tasks/dqFlow.cxx

```diff
@@ -174,53 +174,51 @@ struct DQEventQvector {
     fFC->Initialize(oba, axisMultiplicity, 10);
     delete oba;
 
-    int pows[] = {3, 0, 2, 2, 3, 3, 3};
-    int powsFull[] = {5, 0, 4, 4, 3, 3, 3};
     // Define regions of positive and negative eta in order to create gaps
-    fGFW->AddRegion("refN", 7, pows, fConfigCutEtaMin, fConfigEtaLimitMin, 1, 1);
-    fGFW->AddRegion("refP", 7, pows, fConfigEtaLimitMax, fConfigCutEtaMax, 1, 1);
-    fGFW->AddRegion("full", 7, powsFull, fConfigCutEtaMin, fConfigCutEtaMax, 1, 2);
+    fGFW->AddRegion("refN", fConfigCutEtaMin, fConfigEtaLimitMin, 1, 1);
+    fGFW->AddRegion("refP", fConfigEtaLimitMax, fConfigCutEtaMax, 1, 1);
+    fGFW->AddRegion("full", fConfigCutEtaMin, fConfigCutEtaMax, 1, 2);
```

Comment:
@iarsene These changes were needed because the Generic Framework interfaced has been slightly modified